<?xml version="1.0"?>

<launch>
    <!--Make sure that the drone name here matches the drone name in the simulate.launch file-->
    <arg name="drone_name" default="dji_m210"/>

    <group ns="$(arg drone_name)">
        <node pkg="jellyfish_sim" name="fake_dji_sdk" type="fake_dji_sdk.py" respawn="false" output="screen"/>
    </group>
</launch>

